JavaAssignment/src/umesh/AreaRecAndCircle.java

```diff
@@ -0,0 +1,18 @@
+package umesh;
+
+class AreaRecAndCircle {
+	public static void main(String args[]) {
+		int l = 10;
```

Comment:
write logic in different methods, do not write all the code directly in main method